src/parallax/fourth/card/card.module.scss

```diff
@@ -0,0 +1,40 @@
+:root {
+  --push-down: 0;
+}
+
+// Styles
+.card {
+  border-radius: 10px;
+  max-width: 240px;
+  margin: 0;
+  .img {
```

Comment:
Why so much nesting? In CSS modules all these selectors can be flat, no need to over-complicate.